CONTRIBUTING.md

```diff
@@ -147,44 +162,44 @@ Style guidelines for commit messages are as follows:
 ## Share a Topic ##
 
 When a topic is ready for review and possible inclusion, share it by
-pushing to a fork of your repository in GitLab. Be sure you have registered
-and signed in for [GitLab Access] and created your fork by visiting the
-main [Viskores GitLab] repository page and using the "Fork" button in the
+pushing to a fork of your repository in Github. Be sure you have registered
+and signed in for [Github Access] and created your fork by visiting the
+main [Viskores Github] repository page and using the "Fork" button in the
 upper right.
 
-[Viskores GitLab]: https://github.com/Viskores/viskores
+[Viskores Github]: https://github.com/viskores/viskores
 
 1.  Checkout the topic if it is not your current branch:
 
         $ git checkout my-topic
 
-2.  Push commits in your topic branch to your fork in GitLab:
+2.  Push commits in your topic branch to your fork in Github:
 
-        $ git gitlab-push
+        $ git github-push
```

Comment:
Does this command now exist? It looks like you deleted the `git-gitlab-push` script, and I don't see it replaced with anything.